docs/guide/migrating-to-version-9.md

```diff
@@ -224,6 +251,41 @@ Component has been renamed to `KDropdown`
 
 ### KMultiselect
 
+#### Props
+
+* `testMode` prop has been removed
+* `filterFunc` prop name has been changed to `filterFunction`
+* `disabledTooltipText` property of object passed through `items` prop is no longer supported
+* default value of `collapsedContext` prop has changed to `true`
+* default value of `selectedRowCount` prop has changed to `1`
+
+#### Events
+
+* `item:added` event was renamed to `item-added`
+* `item:removed` event was renamed to `item-removed`
+
+#### Structure
+
+* `k-multiselect-item` class has been changed to `multiselect-item`
+* `k-multiselect-item-container` class has been changed to `multiselect-item-container`
+* `k-multiselect-item-label` class has been changed to `multiselect-item-label`
+* `k-multiselect-group-container` class has been changed to `multiselect-group-container`
+* `k-multiselect-group-title` class has been changed to `multiselect-group-title`
+* `k-multiselect-label` `testid` data attribute has been changed to `multiselect-label`
```

Comment:
In this section where you're calling out the `testid` data attribute, can you please document it as "`data-testid` attribute" instead to make it very clear?